### PR TITLE
fix:fix webhook resource

### PR DESCRIPTION
**Pull request by haozheng95**

### CODE REVIEW

**Comment by haozheng95** [comment]
fix webhook delete other resources

File: bitfusion_device_plugin/webhook/pkg/webhook/webhook.go [hunk 1]
```diff
@@ -374,13 +361,85 @@ func updateAnnotation(target map[string]string, added map[string]string) (patch
 	return patch
 }
 
+func updateInitContainersResources(target, added []corev1.Container) []corev1.Container {
+	maxCpu := zeroQuantity
```
Comment (4everming) [inline bitfusion_device_plugin/webhook/pkg/webhook/webhook.go:363]: the quantity of cpu and memory is set to 0 is valid which means they can have unlimited resource.

**Comment by 4everming** [comment]
LGTM Please fix the conflict

**Comment by 4everming** [comment]
Deprecate this PR. Merged in another PR.

**Pull request closed without merging**